sentry_sdk/client.py

```diff
@@ -330,7 +330,12 @@ def capture_event(
         if session:
             self._update_session_from_event(session, event)
 
-        self.transport.capture_event(event_opt)
+        if event.get("type") == "transaction":
+            # Transactions should go to the /envelope/ endpoint.
+            self.transport.capture_envelope(Envelope(items=[event]))
```

Comment:
I found no evidence that this actually sends to `/envelope/`. Too late now, going to sleep.

Reply:
Sending envelopes to the `/envelope/` endpoint is https://github.com/getsentry/sentry-python/pull/730.